storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,36 @@
+[id="persistent-storage-csi-azure"]
+= Azure CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk.
+
+:FeatureName: Azure Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
+
+???need info on how to install???To create CSI-provisioned PVs that mount to Azure storage assets, you must manually enable the Azure CSI Driver Operator and the Azure CSI driver for deployment.
```

Comment:
I suggest commenting out your question so it doesn't display in preview output or inadvertently later in published output. :)